### DISCUSSION TITLE
Where is the variable `isSearchEnabled` for setting `SearchBar`?

### DISCUSSION THREAD

**Discussion by Kevin-free** [danny-avila/LibreChat#3065] [Q&A] [answered]
Hello, where is the variable `isSearchEnabled` for setting `SearchBar`?

I have the same code and environment variables, but the `SearchBar` is not displayed in the local `http://localhost:3080/c/new`, but it is displayed in the production environment. I want to display the `SearchBar` locally, what should I do?

Local environment pictures:
<img width="234" alt="image" src="https://github.com/danny-avila/LibreChat/assets/26422757/015335d7-135b-4c3c-a48d-f40d7781c968">

Prod environment pictures:
<img width="234" alt="image" src="https://github.com/danny-avila/LibreChat/assets/26422757/a2ea316a-6c22-411c-9caa-0e9c623841d1">

**Comment by Kevin-free** [comment]
Is it because my local environment is running through npm and meilisearch is not running?

**Reply by fuegovic** [reply, marked as answer]
You do need to have meilisearch running for search to show up
